pkg/proc/proc_test.go

```diff
@@ -586,6 +586,7 @@ func TestNextGeneral(t *testing.T) {
 }
 
 func TestNextConcurrent(t *testing.T) {
+	skipOn(t, "broken", "freebsd")
```

Comment:
A bunch of these have been reintroduced due to merge errors. In fact a lot of changes in this file seem to be merge errors and should be rolled back.

Unless maybe there are no merge errors and github UI is just lying to me because of the merge conflict.